src/showcase/stories/angular/sidebar/icon-sidebar-link/icon-sidebar-link.stories.ts

```diff
@@ -19,10 +19,11 @@ const argTypes: ArgTypes = {
 };
 
 const args: Args = {
-  'icon-name': 'glass-cocktail-small',
+  iconName: 'glass-cocktail-small',
   'accessibility-label': 'Go to the party',
   href: '#',
   currentPage: false,
+  target: '_blank',
```

Comment:
do we need this as default? I think it would rarely be used

Reply:
i tend to test thing by clicking on them, and it makes me angry when the page changes under my eyes :D

Reply:
I understand :D. It's just that if it gets copied, they have an unusual default.